src/test/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAssociationManagerTest.java

```diff
@@ -378,8 +379,8 @@ public final void userAssociationManager_NoPermissionUserHasNoValidAssociation_f
 	@Test
 	public final void userAssociationManager_TokenMustBeSixCharactersAndRandom()
 			throws SegueDatabaseException, UserGroupNotFoundException {
-		UserAssociationManager managerUnderTest = new UserAssociationManager(dummyAssociationDataManager,
-				dummyGroupDataManager);
+		UserAssociationManager managerUnderTest = new UserAssociationManager(
+				dummyAssociationDataManager, dummyUserManager, dummyGroupDataManager);
 		Long someAssociatedGroupId = 5654811L;
 
 		RegisteredUserDTO someUserRequestingAccess = createMock(RegisteredUserDTO.class);
```

Comment:
Might be worth adding a test to see if event managers can get /post things they shouldn't be able to?